### PR TITLE
Wrong week number is displayed

### PR DESCRIPTION
**Pull request by Coenego**
The wrong academic week number seems to be displayed in the student UI.

Steps to reproduce:
- Navigate to a term using the term navigator => 'Week 2' should be 'Week 1'
- Go backward and forward one week => 'Outside term' should be 'Week 1'

screen shot 2015-02-23 at 10 08 33

### CODE REVIEW

**Comment by Coenego** [comment]
Assigning to @bp323 for review

File: shared/gh/js/constants/gh.constants.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ define(['exports'], function(exports) {
 
         // Keep track of number of milliseconds in a day, week and month for use in the calendar
         'PERIODS': {
+            'hour': 1000 * 60 * 60,
```
Comment (bp323) [inline shared/gh/js/constants/gh.constants.js]: Looks like this is not being used.

File: tests/qunit/tests/js/api.util.js [hunk 2]
```diff
@@ -173,6 +173,10 @@ require(['gh.core', 'gh.api.tests'], function(gh, testAPI) {
 
         // Verify that a valid week number is returned when specifying an out-of-term date that leans close to the start of a term
         weekNumber = gh.utils.getAcademicWeekNumber(gh.utils.convertISODatetoUnixDate('2015-01-12T10:30:00.000Z'));
+        assert.strictEqual(weekNumber, 1, 'Verify that a valid week number is returned when specifying an out-of-term date');
+
+        // Verify that a valid week number is returned when specifying an out-of-term date that leans close to the start of a term, using precise
+        weekNumber = gh.utils.getAcademicWeekNumber(gh.utils.convertISODatetoUnixDate('2015-01-12T10:30:00.000Z'), true);
         assert.strictEqual(weekNumber, 0, 'Verify that a valid week number is returned when specifying an out-of-term date');
```
Comment (bp323) [inline tests/qunit/tests/js/api.util.js]: Can you add to the description of the assert that it's testing the `precise` parameter?

**Comment by bp323** [comment]
Looks good! Only 2 minor comments remaining. Back to @Coenego

**Comment by bp323** [comment]
Verified and merged.

**Pull request merged**